release_build_files/readme.md

```diff
@@ -633,6 +633,8 @@ code.
 ## Release Notes
 ### Upcoming Release
 -   Changes
+    - General (iOS): Minimum iOS deployment target is now 13.0.
+    - General (tvOS): Minimum tvOS deployment target is now 11.0.
```

Comment:
Just double checking, is this number correct?

Reply:
No, I actually had changed it to 11 and then decided to change it back since we already had it as 12 in some places.